packages/gatsby/src/redux/index.ts

```diff
@@ -129,18 +129,22 @@ export const saveState = (): void => {
   })
 }
 
-export const saveStateForWorkers = (slices: Array<GatsbyStateKeys>): void => {
+export const savePartialStateToDisk = (
```

Comment:
Renamed the functions to make them "usable" both for main process and worker process